keyboards/qd60rgb/readme.md

```diff
@@ -0,0 +1,12 @@
+# qd60rgb pcb with rgb 
```

Comment:
Could this file be aligned with the template <https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md>, mainly an image and hardware availability link?

Reply:
Keyboard Maintainer:[demos]"https://github.com/liudongxin"
Hardware Supported:Demo Studio qd60rgb
like this? haven't run a gb

Reply:
Generally, personal and small run projects would be asked to relocate to the `handwired` directory. Are there intentions to run a group buy?

Reply:
maybe a small run,i'm not sure,it's the first time i use it